webapp/components/DeleteData/DeleteData.spec.js

```diff
@@ -68,15 +66,15 @@ describe('DeleteData.vue', () => {
       jest.clearAllMocks()
     })
 
-    it('defaults to deleteContributions to false', () => {
+    it('checkbox deleteContributions defaults be false', () => {
       expect(wrapper.vm.deleteContributions).toEqual(false)
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-27T09:18:05Z" title="Wednesday, May 27th 2020, 11:18:05 am +02:00">May 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -93,7 +93,11 @@ describe('DeleteData.vue', () => {
         deleteAccountBtn = wrapper.find('[data-test="delete-button"]')
       })
 
-      it('if deleteEnabled is true and only deletes user by default', () => {
+      it('if deleteEnabled is true and only deletes user ', () => {
+        mocks.$t.mockImplementation(() => deleteContributionsMessage)
```


As far I see you don’t need this one and all the others `t.mockImplementation` calls. 👇🏼
For debugging use in the mock definition
```
    mocks = {
      $t: jest.fn((a) => a),
```
instead to have the texts identifier returned.